Backend/dev/migrations/dynamic-offer-driver-app/0172-special-location-tag.sql

```diff
@@ -0,0 +1,140 @@
+ALTER TABLE atlas_driver_offer_bpp.quote_special_zone ADD column special_location_tag text;
+ALTER TABLE atlas_driver_offer_bpp.estimate ADD column special_location_tag text;
+ALTER TABLE atlas_driver_offer_bpp.driver_quote ADD column special_location_tag text;
+ALTER TABLE atlas_driver_offer_bpp.search_request ADD column special_location_tag text;
+ALTER TABLE atlas_driver_offer_bpp.fare_policy ADD column "description" text;
+CREATE TABLE atlas_driver_offer_bpp.special_location_priority (
+  id character(36) NOT NULL PRIMARY KEY,
+  merchant_id character(36) NOT NULL REFERENCES atlas_driver_offer_bpp.merchant (id),
+  category character varying(255) NOT NULL,
+  pickup_priority integer NOT NULL,
+  drop_priority integer NOT NULL
+);
+CREATE INDEX idx_special_location_priority ON atlas_driver_offer_bpp.special_location_priority USING btree (merchant_id, category);
+-- All Category Priorities for different supported locations based on MerchantId
+INSERT INTO atlas_driver_offer_bpp.special_location_priority VALUES ('f0e77e85-11fa-4949-af8e-92d8b915e3d9','favorit0-0000-0000-0000-00000favorit','SureMetro','2','2');
```

Comment:
Comment all this code and remove commented code. All this should go inside seed files

Reply:
I have removed hardcoded merchantId so now it's no longer a hardcoded migration that will only work with dev environment. Now it's fine right ?